data/json/bionics.json

```diff
@@ -167,12 +168,10 @@
     "name": { "str": "Subdermal Carbon Filament" },
     "description": "Lying just beneath your skin is a thin armor made of carbon nanotubes.  This reduces bashing damage by 2 and cutting damage by 4.",
     "occupied_bodyparts": [
-      [ "torso", 5 ],
+      [ "torso", 8 ],
       [ "head", 2 ],
       [ "arm_l", 2 ],
       [ "arm_r", 2 ],
-      [ "hand_l", 1 ],
-      [ "hand_r", 1 ],
```

Comment:
It's armor. Putting armor on hand and providing support for so many small joints would be encumbering, which for low protection if offers would be detrimental.
Hands slots are sort of premium considering how many other CBMs uses them for better effect.

Reply:
I can see an equivalent argument for plating, but this is rather more microscale anyway from the description of it. OTOH, I can also see the argument regarding hands slots - pity that partial slots are currently not possible IIRC, since I doubt this would take up an entire one. (OTOH, if the injector is still using a needle in one hand, a partial slot in the hand would be appropriate for it.)

Reply:
Reverted